deploy/dex_structure.py

```diff
@@ -1814,9 +1851,12 @@ def staking_deploy(self, contracts_index: str, deployer_account: Account, networ
             if not network_providers.check_simple_tx_status(tx_hash, "set rewards per block in stake contract"): return
 
             if version == StakingContractVersion.V3Boosted:
+                deployed_staking_contract.resume(deployer_account, network_providers.proxy)
```

Comment:
`tx_hash = deployed_staking_contract.resume(deployer_account, network_providers.proxy)`

Reply:
removed completely since the need to resume was removed from contract